doc(setting): eventual danger zone setting update (#858)
longhorn/website: content/docs/1.6.0/advanced-resources/deploy/storage-network.md

Applying the setting will add `k8s.v1.cni.cncf.io/networks` annotation and recreate all existing instance-manager, and backing-image-manager pods.
Longhorn will apply the same annotation to any new instance-manager, backing-image-manager, and backing-image-data-source pods.

> > **Warning**: Do not suggest changing this setting with volumes attached.

Reviewer: ```suggestion
>  **Warning**: Do not change this setting while volumes are still attached.
```

> > **Warning**: To apply a setting immediately, stop all workloads and detach all Longhorn volumes before changing the setting.

Reviewer: ```suggestion
> **Important**: To ensure that your preferred settings are immediately applied, stop all workloads and detach all Longhorn volumes before configuring the settings.
```

Author: Updated. Thanks

>
> When all volumes are detached, Longhorn attempts to restart all Instance Manager and Backing Image Manager pods to apply the setting.

> > When one or more volumes are attached, the customized setting will be applied to the Instance Manager without engine and replica instances running, and you are required to reconfigure this setting when all volumes are finally detached.

Reviewer: ```suggestion
> When one or more Longhorn volumes are still attached, the customized settings are applied to the Instance Manager only when no engines and replica instances are running. You might be required to reconfigure the settings after detaching the remaining volumes.
```

Author: Modified. Thanks

> **Warning**: Do not change this setting with volumes attached.
>

> > Longhorn will try to restart all instance-manager, and backing-image-manager pods if all volumes are detached and eventually restart the instance manager pod without instances running on the instance manager..

Reviewer: ```suggestion
> When all volumes are detached, Longhorn attempts to restart all Instance Manager and Backing Image Manager pods.
```

Reviewer: > eventually restart the instance manager pod without instances running on the instance manager.

This is unclear. Do you mean that the pod is restarted only when no instances are running on the Instance Manager?

Author: Yes, it is.
The instance manager pod will be restarted only when no instances are running on the Instance Manager and the setting is not applied to the pod.

Reviewer: Thanks. I have updated the suggestion.

Author: Modified. Thanks.
